packages/pages/src/components/link/methods.ts

```diff
@@ -20,27 +20,6 @@ const getHref = (cta: CTA): string => {
   return cta.link;
 };
 
-/**
- * CTA constructor
- *
- * @param {string} link
- * @param {LinkType} linkType
- * @param {string} label
- *
- * @returns {boolean}
- */
-export function cta(
```

Comment:
is this constructor not needed anymore?

Reply:
@nmanu1 I replaced it with `makeCTA` inside the component. One so the name doesn't conflict and two it now takes the LinkProps so theres no real value for it existing outside the react component

Reply:
I see. but this wasn't used before the refactor either from what I can see. was it just not being used anywhere? and it's only available internally (not publicly exported), right?

Reply:
It was exported to be used outside of the component but now it is done in the component. I also didn't like the name of the function being the same as the type